test/e2e/demos_test.go

```diff
@@ -0,0 +1,34 @@
+// This module is meant to cover all the demos as we show them. To make it as close
```

Comment:
Why this file belongs to this commit?

Reply:
It tests the tutorial, following the same steps and using same examples to verify that it indeed works and we don't make fools of ourselves in the docs.